unit_test/transforms/test_mapper.py

```diff
@@ -0,0 +1,34 @@
+#  Copyright (c) 2023. Deltares & TNO
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import unittest
+from simulator_core.entities.esdl_object import EsdlObject
+from simulator_core.infrastructure.utils import pyesdl_from_file
+from simulator_core.adapter.transforms.mappers import EsdlEnergySystemMapper
+
+
+class EsdlEnergySystemMapperTest(unittest.TestCase):
+
+    def test_to_entity(self):
+        # act
+        esdl_file_path = r'.\testdata\test1.esdl'
+        esdl_object = EsdlObject(pyesdl_from_file(esdl_file_path))
+        # arrange
+        network = EsdlEnergySystemMapper().to_entity(esdl_object)
+        # assert
+        self.assertTrue(True)
```

Comment:
Why this assert statement?

Reply:
I believe you want to check whether you have created a network asset?

Reply:
Adjusted test